eclipsesource/theia
# fix: open ai messages

#### What it does

Fixes an edge case in message handling for open ai

#### How to test

- post a question to see whether chat is working
- remove all models in open ai settings
- post another question in same thread, which triggers an error
- reset all models
- post another question in same thread

The LM should properly answer. Previously there was an error as a message sent to OpenAI API was `null`, resulting in a 400 error

#### Follow-ups



#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
